metric/metric.go

```diff
@@ -46,3 +48,9 @@ type PolicyCountMetric struct {
 	EnvironmentName string
 	Count           int
 }
+
+// TaskCountMetric is the API message for database count metric
+type TaskCountMetric struct {
+	TaskType api.TaskType
+	Count    int
```

Comment:
Should we also count the task status? (The [Product Metrics List](https://bytebase.feishu.cn/sheets/shtcnYNIA6eWcKGdfzLq3P4afUf?sheet=uR5k3v&useNewLarklet=1) doc doesn't mention this @tianzhou